ark/utils/load_utils.py

```diff
@@ -142,7 +146,7 @@ def load_imgs_from_tree(data_dir, img_sub_folder=None, fovs=None, channels=None,
         channels = [chan for _, chan in sorted(zip(channels_indices, channels))]
 
     if len(channels) == 0:
-        raise ValueError("No images found in designated folder")
+        raise ValueError("No channels specified")
```

Comment:
So the reason this one says no images found in directory is because when channels = None, channels get populated by whatever images are in each tree. If there are none, its empty, hence the warning. The one I commented on above doesn't follow that logic, so the warning should be modified